src/app/components/BibPage/BibPage.jsx

```diff
@@ -206,34 +221,25 @@ class BibPage extends React.Component {
     const bNumber = record && record.idBnum ? record.idBnum : '';
     const marcRecordLink = bNumber ? 'https://catalog.nypl.org/search~S1?' +
       `/.b${bNumber}/.b${bNumber}/1%2C1%2C1%2CB/marc` : '';
+    const searchURL = createAPIQuery({});
```

Comment:
We do need the function. It's necessary to pass it down as a function and we need to call it to get the current url for this component. Also, basicQuery is being called directly but it's assigned to createAPIQuery.